apps/src/templates/sessions/LockoutPanel.jsx

```diff
@@ -227,7 +233,7 @@ LockoutPanel.propTypes = {
   deleteDate: PropTypes.instanceOf(Date).isRequired,
   pendingEmail: PropTypes.string,
   requestDate: PropTypes.instanceOf(Date),
-  studentEmail: PropTypes.string.isRequired,
+  disallowEmail: PropTypes.string.isRequired,
```

Comment:
Should this be "disallowedEmail"? To me, "disallowEmail" feels like it should be a boolean deciding whether to disallow the email.

Reply:
fair enough! I'll make that change